src/pages/AuthenticationPages/Signup.jsx

```diff
@@ -0,0 +1,116 @@
+import React, { useState } from "react"
+import "./UserAuth.css"
+import { Link, useNavigate } from "react-router-dom"
+import axios from "axios"
+import { useToast } from "../../context/index"
+
+function Signup()
+{
+    const { showToast } = useToast()
+
+    const [termsAndConditionsCheckbox, setTermsAndConditionsCheckbox] = useState(false)
+    const [newUserName     , setNewUserName]     = useState('')
+    const [newUserEmail    , setNewUserEmail]    = useState('')
+    const [newUserPassword , setNewUserPassword] = useState('')
```

Comment:
Suggestion: you can also use object of `user` with key value pair of email, name and password

Reply:
Yeah I think this is fine because in that case I would have to destructure it again and access properties using dot operator.